### PR TITLE
Adding pipenv support

### PR DESCRIPTION
**Pull request by afs2015**
This adds pipenv support for development of the homepage.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -20,6 +20,13 @@ If you want to test out a change before you submit it, download and install
 
     $ pip install lektor
 
+If you use `pipenv`_, a .Pipfile is provided that has Lektor and Python 2.7
```
Comment (goanpeca) [inline README.rst]: Hi @afs2015 now that we are adding this, maybe it is a good idea to include a couple of subsections

Installing development environment
------------------------------------

Using pip
~~~~~~~


Using pipenv
~~~~~~~~~

**Comment by afs2015** [comment]
@goanpeca: Great feedback, i've added changes that incorporates the subsections you recommended. 

On an unrelated note do we want to edit the LICENSE file to have the copyright be 2018?

**Comment by afs2015** [comment]
Adding a commit, just realized I forgot to move the pipenv run server to the local server section.

**Comment by goanpeca** [comment]
Thanks for the changes @afs2015 :-), I am sure they will improve users trying to help with the website.

**Comment by afs2015** [comment]
No problem, @goanpeca! I think I cleaned up the last thing in the README file. Anything else I need to do for this pull request? 

I'm new to contributing to BeeWare and am still getting the hang of this process.

**Comment by danyeaw** [comment]
@afs2015 Thanks for submitting this pull request. Due to issues @freakboy3742 has been having with the stability of pipenv on macOS, we are not ready yet to endorse the tool as part of the development environment for BeeWare projects.

**Comment by afs2015** [comment]
After talking with @danyeaw at the PyCon 2018 sprint, we want to hold on this until pipenv is more stable for OS X. I'll keep this pull request open and merge in changes from master until pipenv gets the green light.

**Pull request closed without merging**